### ISSUE TITLE
Add author / last updated by info to a post

### ISSUE THREAD

**Issue by fergusfrl** [fergusfrl/RiverWiki#23]

**Comment by pcrossnz** [comment]
Agreed. This will be good to have. Is there any way that we can easily have the previous description saved in the background (version control). Like most Wiki's you can see how changes to the description evolve over time. I know this might be a big ask

**Comment by fergusfrl** [comment]
Will need to have a think re. design for version control, I think it's worth creating a new issue for that feature.
